bin/storm.py

```diff
@@ -241,7 +240,6 @@ def jar(jarfile, klass, *args):
             extrajars=[tmpjar, USER_CONF_DIR, STORM_BIN_DIR],
             args=args,
             daemon=False,
```

Comment:
Ah... that makes sense now. So how concerned are we about leaking a tmp file vs. allowing users to build tools around topology submission (and get the right exit code)?

Would it make sense to have a separate command to do the topology transform instead of in-lining it with topology submission? I.e. two steps: 1. transform topology, 2. submit topology jar output from step 1.